src/components/ZclEndpointCard.vue

```diff
@@ -419,6 +419,15 @@ export default {
       if (this.$route.path !== '/') {
         this.$router.push({ path: '/' })
       }
+      if (this.$store.state.zap.isMultiConfig) {
+        if (categoryTmp === 'zigbee') {
```

Comment:
i think we should use helperCategory in this condition
dbEnum.helperCategory.matter
dbEnum.helperCategory.zigbee

Reply:
thank you, fixed